tests/test_syntax/extensions/test_abbr.py

```diff
@@ -119,21 +120,67 @@ def test_abbr_multiple_nested(self):
             )
         )
 
-    def test_abbr_override(self):
+    def test_abbr_glossary(self):
```

Comment:
I understand why the new test was added, but why was the existing test removed/replaced? Shouldn't we have both tests?

Reply:
Restored. I forgot that I didn't add that test and accidentally removed it when removing artifacts from `use_last_abbr`.